BWMatch.py

```diff
@@ -0,0 +1,139 @@
+'''
+Implement efficient algos from Pevzner & Compeau BWT chapter
+Construct BWT from suffix tree
+Suffix tree:all nodes are labeled with the start and end locations in T corresponding to their text string
+Leaf nodes are additionally labeled with the index of where the suffix ending there originated in T
+Then to find the BWT, we do a DFS where we always start by exploring the child node with the earlier character in the alphabet. And when we get to a leaf, we append T[i - 1] to the growing BWT.

```

Comment:
I'd wrap your lines at 80 characters or so (this gets lumped in with other pep-8 stuff).